sources/examples/README.rst

```diff
@@ -0,0 +1,39 @@
+OpenDylan example projects
+==========================
+
+Each sub-directory contains a standalone Dylan project. To compile them, you should have 
+a `working OpenDylan <https://opendylan.org/download/index.html>`_ installation. 
+
+To verify that your installation is working correctly, try checking the compiler's version::
+
+    $ dylan-compiler -version
+    Version 2019.1
+
+If a version string isn't appearing, ``dylan-compiler`` may not be on your ``$PATH``. For 
+help, check the `installation instructions <https://opendylan.org/download/index.html>`_ again 
+or reach out on one of the `community's communication channels <https://opendylan.org/community/index.html>`_.
+
+
+Compiling examples
+------------------
+
+To compile any of the examples, run ``dylan-compiler`` with the ``-build`` option. For any given project, 
+follow this is the pattern::
```

Comment:
following this pattern::

Reply:
Should be fixed. I've cut this sentence out.